src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    //fields
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static int counter = 0;
+
+    //utility methods
+    public static void printWithDivider(String stringWithinDivider) {
+        String breakLine = "\t____________________________________________________________";
+        System.out.println(breakLine);
+        stringWithinDivider = stringWithinDivider.replace("\n", "\n\t");
+        System.out.println("\t" + stringWithinDivider);
+        System.out.println(breakLine);
+    }
+    public static ArrayList<String> processInput(String line, String delimiter) {
+        ArrayList<String> words = new ArrayList<>();
+        int divider = line.indexOf(delimiter);
+
+        words.add(line);
+
+        if (divider != -1) {
+            words.set(0, line.substring(0, divider));
+            words.add(line.substring(divider+1));
+        }
+        return words;
+    }
+
+    //command methods
+    public static void hello() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        printWithDivider("Hello! I'm Duke\nWhat can I do for you?");
+    }
+    public static String numerateList(ArrayList<Task> list) {
+        String output = "";
+        int number = 1 ;
+        for (Task item : list) {
+            output += String.format("%d. %s", number, item.toString());
+            if (number != list.size()) {
+                output += "\n";
+            }
+            number ++;
+        }
+        return output;
+    }
+    public static void addTask(String type, String description) {
+        Task toBeAdded;
+        ArrayList<String> words = processInput(description, "/");
+        String taskName = words.get(0);
+        String addInfo = words.size() >= 2 ? words.get(1) : null;
+
+        switch (type) {
+        case "todo":
```

Comment:
Can consider extracting out some methods or sub-methods to a new class (eg. TaskManager, TaskList) to manage any functions related to Task